apps/betterangels-backend/notes/migrations/0014_note_migrate_title_to_purpose.py

```diff
@@ -0,0 +1,23 @@
+# Generated by Django 5.1.2 on 2024-11-05 21:04
+
+from django.db import migrations
+from django.db.models.query import Q
+
+
+def migrate_title_to_purpose(apps, schema_editor):
+    Note = apps.get_model("notes", "Note")
+    notes = Note.objects.exclude(Q(purpose__isnull=True) | Q(purpose__exact=""))
+    for note in notes:
+        note.purpose = note.title
```

Comment:
@mikefeldberg -  just curious, is title going to get deprecated/removed in a follow up ticket? 
If not then is the title column going to get wiped?

Reply:
`title` is getting removed. 🤔 coulda sworn we had a cleanup ticket for it but I don't see it so I made one https://betterangels.atlassian.net/browse/DEV-1001